tests/std/tests/VSO_0157762_feature_test_macros/test.compile.pass.cpp

```diff
@@ -615,6 +615,20 @@ STATIC_ASSERT(__cpp_lib_erase_if == 202002L);
 #endif
 #endif
 
+#if _HAS_CXX20 && !defined(__EDG__)

```

Comment:
This block be after `__cpp_lib_filesystem`.

Reply:
whoops, used same order as yvals.